lib/Runtime/Language/Arguments.h

```diff
@@ -84,8 +92,19 @@ inline int _count_args(const T1&, const T2&, const T3&, const T4&, const T5&, Js
 #define CALL_ENTRYPOINT_NOASSERT(entryPoint, function, callInfo, ...) \
     entryPoint(function, callInfo, ##__VA_ARGS__)
 #elif defined (_ARM64_)
+#ifdef __linux__
+// Linux ARM64 uses AAPCS64: first 8 args in x0-x7, rest via stack.
+// Fill x2-x7 with nulls here to force the expected stack layout:
+// [RetAddr] [function] [callInfo] [args...]
+#define CALL_ENTRYPOINT_NOASSERT(entryPoint, function, callInfo, ...) \
```

Comment:
I'd expect this to break the JavascriptStackWalker, I saw you said it was "fragile", what's the status?